JP>EN Tutorial Translation for 「ダッシュパネル」について "Dash Panels"

SanseiMuramasa opened the pull request:
As a side note, this tutorial is related to "Enemy Location" because that tutorial also references "dash panels".

SanseiMuramasa committed:
JP>EN Tutorial Translation for 「ダッシュパネル」について "Dash Panels"

As a side note, this tutorial is related to "Enemy Location" because that tutorial also references "dash panels".

alama commented:
You  need to move the file out of the Files folder

SanseiMuramasa commented:
I will try that shortly.

SanseiMuramasa committed:
Moved it out of "Files"

SanseiMuramasa committed:
Moved it to "Tutorial/Window"

alama commented:
Hehe; note to self: Do not allow CSV files on the root of repos

Can you move the file to "Tutorial/Window" ?

SanseiMuramasa commented:
It has been moved to "Tutorial/Window".

SanseiMuramasa committed:
Slight modification to interpretation of words

SanseiMuramasa committed:
Changed translation of 乗り継ぎ "combo/connect" to include "dash panel rings"

PLEASE READ:

The original JP translation does not mention dash panel rings (yet they exist in PSO2).

So I feel it should be included in the EN translation in order to make the tutorial more accurate.

What are dash panel rings? The dash panel rings you go through on stages like Tokyo Exploration.

乗り継ぎ implicity implies it's the rings that are being used to combo but this is sorta lost in translation in English

So I had to explicitly mention the presence of rings (even if the Japanese translation doesn't explicitly mention them as to communicate the actual ideas of "Dash Panels" properly in English.

SanseiMuramasa commented:
PLEASE READ:

(Why does this translation mention dash panel rings, yet the JP text does not mention them?)

The original JP translation does not mention dash panel rings (yet they exist in PSO2).

So I feel it should be included in the EN translation in order to make the tutorial more accurate.

What are dash panel rings? The dash panel rings you go through on stages like Tokyo Exploration.

乗り継ぎ implicity implies it's the rings that are being used to combo but this is sorta lost in translation in English

So I had to explicitly mention the presence of rings (even if the Japanese translation doesn't *explicitly* mention them) as to communicate the actual ideas of "Dash Panels" properly in English.

SanseiMuramasa committed:
A more clear and simple translation/explanation of dash panels

SanseiMuramasa commented:
This is my final, simplified translation of dash panels.

I cleared up all the mess and tried to present the idea of "dash panels" while including an explanation of dash panel rings (乗り継ぎ) in English.

Feel free to proofread any mistakes of my final, simplified version.

ZhugeSong commented on Tutorial/Window/co_040000.csv:
```diff
@@ -0,0 +1,3 @@
+Title#0,"""Dash Panels"""
+text01#0,"""Press the<%Access> to access the dash panels.<br>This will allow you to dash for a limited time."""
+text02#0,"""While dashing, every time you jump through another dash panel ring,<br>you can dash for longer periods of time.<br>(Re-accessing the dash panels won't be neccessary if you jump through the dash panel rings.）<br><br>(Peformable actions while dashing)<br>Press<%Jump>to jump.<br>Press<%NormalAttack>/<%PhotonArts>to kick.<br><br>The attack power of your kick<br>can be enhanced up to a max ...
```
Noticed misspelling at end of line 3: "immediately"

SanseiMuramasa replied:
Will fix ASAP.

ZhugeSong commented on Tutorial/Window/co_040000.csv:
```diff
@@ -0,0 +1,3 @@
+Title#0,"""Dash Panels"""
+text01#0,"""Press the<%Access> to access the dash panels.<br>This will allow you to dash for a limited time."""
```
Should this really say "Press the <%Access> to[...]"? I assume that variable finds the keybinding the player has set, so this would result in "Press the E to[...]". So I would think it should say "Press <%Access> to[...]" instead.

SanseiMuramasa replied:
I assumed that's a variable that the player set in the Settings.

(Whatever the player set the key for "Access")

So... I didn't touch the variable.

SynthSy replied:
> Should this really say "Press the <%Access> to[...]"? I assume that variable finds the keybinding the player has set, so this would result in "Press the E to[...]". So I would think it should say "Press <%Access> to[...]" instead.

You are correct.

SanseiMuramasa replied:
I didn't realize s/he was pointing out the word order.

I'll fix that right away... sorry I didn't notice the first time.

SanseiMuramasa committed:
"Immediately" Spelling correction.

SanseiMuramasa commented:
The spelling of "immediately" has been corrected.

SanseiMuramasa committed:
Coding fix

SanseiMuramasa committed:
Omitted the word "the" in Line 2

SanseiMuramasa committed:
Added  in spaces for <%KeyVariables>

SanseiMuramasa committed:
Removed <br> linebreaks and organized into paragraphs.

SanseiMuramasa committed:
Inserted 1 more double line break to organize it into 4 paragraphs (based on the original JP context)

SanseiMuramasa commented:
Oh... so the PR has to be open in order for someone to merge it... never mind.

SanseiMuramasa commented:
I'm confident with this final translation feel free to proofread for more mistakes and merge it when it's ready to go.
